src/server.h

```diff
@@ -1580,14 +1584,18 @@ struct redisCommand {
     int firstkey; /* The first argument that's a key (0 = no keys) */
     int lastkey;  /* The last argument that's a key */
     int keystep;  /* The step between first and last key */
-    long long microseconds, calls;
+    long long microseconds, calls, rejected_calls, failed_calls;
     int id;     /* Command ID. This is a progressive ID starting from 0 that
                    is assigned at runtime, and is used in order to check
                    ACLs. A connection is able to execute a given command if
                    the user associated to the connection has this command
                    bit set in the bitmap of allowed commands. */
 };
 
+struct redisError {
+    long long count;
+};
```

Comment:
Do we intend to extend this? Otherwise it would be easier to just store the count in the rax.

Reply:
@madolson I tought of it as wrapper struct for future requirements.

Reply:
Okay, as long as that is intended, it's okay.